src/frequenz/sdk/config/_config.py

```diff
@@ -12,6 +12,7 @@
 _logger = logging.getLogger(__name__)
 
 T = TypeVar("T")
+"""Type variable for validating config values."""
```

Comment:
We can keep this but in general I wouldn't document `TypeVar`s unless there is something interesting to say about them, also ingeneral they should be private.